src/App.tsx

```diff
@@ -103,17 +109,25 @@ export default function App() {
             />
           </Grid>
         </Grid>
-        <Grid container justify="flex-end">
+
+        <Grid container justify="space-between">
```

Comment:
```suggestion
        <Grid container justify="flex-end">
```

I actually think flex-end works best here so the buttons are next to each other instead of opposite end of the screen